pkg/server/server.go

```diff
@@ -127,6 +127,8 @@ func (s *Server) setupRouter() (*httprouter.Router, error) {
 
 	router.GET("/healthz", s.handleHealthz)
 
+	router.GET("/api/v1/server_info", s.getServerInfo)
```

Comment:
For the sake of consistency

```suggestion
	router.GET("/api/v1/server_info", s.handleServerInfo)
```

Reply:
Noted